src/components/ReportActionItem/IOUAction.js

```diff
@@ -61,7 +61,6 @@ const IOUAction = (props) => {
             />
             {shouldShowIOUPreview && (
                 <IOUPreview
-                    pendingAction={lodashGet(props.action, 'pendingAction', null)}
```

Comment:
This is not needed, in fact it's addin more opacity than neede
https://github.com/Expensify/App/blob/fa187d8c8237f6314753f068e3b110422e9a74bc/src/pages/home/report/ReportActionItem.js#L193-L200
already adds the correct opacity to the group.

Reply:
Trying to understand this change.

I am not quite sold on the fact that we don't need to check for pendingAction here whereas in the IOUDetailsModal page you are calculating the same and passing the value to prop.

Taking a deep look at the code `opacity: 0.5` is added only when `props.network.isOffline && props.pendingAction` so if we remove this how will it be shown greyed out?

Does IOUPreview in the reportScreen look greyed out in your case?

<img width="635" alt="Screenshot 2022-12-29 at 7 02 11 PM" src="https://user-images.githubusercontent.com/35863227/209961324-6570b129-ee35-4255-8ca9-00e918adba72.png">

Reply:
I think the `pendingAction` is already set here

https://github.com/Expensify/App/blob/main/src/components/ReportTransaction.js#L69

I think this might need to be refactored? We're wrapping the IOUPreview in OfflineWithFeedback twice, I'm not sure why there could be a good reason for this.

Reply:
the IOUPreview in the reportScreen is already in a view that makes opacity 0.5 when offline and pending, so it was actually being set to 0.25, I found that out while trying out css styles on the component.

Reply:
ReportActionItem uses IOUAction that uses the IOUPreview

Reply:
Actually this is what's responsible for showing the preview as greyed out: 

https://github.com/Expensify/App/blob/main/src/pages/home/report/ReportActionItem.js#L214

@techievivek how did you make the IOUPreview not greyed out while offline, by simply running code from this PR?

Reply:
I think the nested OfflineWithFeedback component are only needed for nested errors, we set the `pendingAction` in `ReportActionItem`, right?

Reply:
> we set the pendingAction in ReportActionItem, right?

I'm not sure I follow 100% what you mean @youssef-lr, we set the pendingAction of the `OfflineWithFeedback` in `ReportActionItem` yes, but `ReportActionItem` has no pendingAction prop

Reply:
and the componets are being greyed out if I create the money request while being offline

Reply:
I mean, I was clarifying to @techievivek that the `OfflineWithFeedback` component responsible for making the action as greyed out, is the one in `ReportActionItem`, so even if we remove `pendingAction` from `IOUPreview` it will still show up as greyed out. So I'm curious what happened in @techievivek's screenshot because it looks like it's not greyed out while he's offline.

Reply:
> @techievivek how did you make the IOUPreview not greyed out while offline, by simply running code from this PR?

Ah, the action actually got synced when I came online and this screenshot is of when I went offline after that.

Alright, I understand the flow now. And make sense to get rid of `pendingAction` otherwise it does 50% of already applied opacity and that's why Andre was seeing 0.25. Thanks


@youssef-lr 
> I think the nested OfflineWithFeedback component are only needed for nested errors, we set the pendingAction in ReportActionItem, right?
Yeah, I am thinking the same.